Sources/HummingbirdCore/Response/ResponseBody.swift

```diff
@@ -70,8 +70,8 @@ public struct ResponseBody: Sendable {
     }
 
     @inlinable
-    public consuming func write(_ writer: any ResponseBodyWriter) async throws {
-        try await self._write(writer)
+    public consuming func write(_ writer: consuming any ResponseBodyWriter) async throws {
```

Comment:
Isn't this supposed to be `inout any ResponseBodyWriter`?

Reply:
In the situations where I could use this as an inout ie tests it doesn't work as I have to pass it as `any ResponseBodyWriter`. You can't pass a concrete version of the protocol as the inout allows you to change the type to any `ResponseBodyWriter`. 

Because the variable is consumied it is mutable within the function that consumed it.